migrations/20211103120345_add_new_group_widgets.js

```diff
@@ -0,0 +1,19 @@
+
+exports.up = function (knex) {
+  return knex('widgets').insert([
+    {id: 11, name: 'nearby_relevant_groups' },
```

Comment:
I wonder if we wait on this until we know exactly which widgets we want/need? since we are not doing farm comparison for now right?

Reply:
The widget list will need to be revisited. Will include in our design chat

Reply:
anything to do here now or shall we merge as is and work on later?